SqlServer/Import-BacpacToSqlServer/Import-BacpacToSqlServer.psm1

```diff
@@ -1,4 +1,4 @@
-<#
+﻿<#
```

Comment:
[Using this script from the contest menu](https://github.com/Lombiq/Utility-Scripts/#:~:text=To%20be%20able,from%20here.) now yields this:
![image](https://user-images.githubusercontent.com/1976647/179863068-803a95d4-da44-49a8-8f8b-42bbb27ea691.png)

Reply:
My bad, looks like the behavior of `Join-Path` is just too different between `pwsh` and `powershell`. I replaced it with `[System.IO.Path]::Combine`, that at least works the same across versions.

Reply:
I still get this with the latest source.

Reply:
You shouldn't get the exact same error, becuase the affected code was replaced with something else. But I've noticed, that the `20..12` range is super tiny, for me the exe is actually in the "160" directory. So I've rewritten that part from scratch to just use `Get-ChildItem` (and select the numeric directory names) instead of using a hard coded integer range. This worked for me for sure:
![image](https://user-images.githubusercontent.com/4007293/180110359-5d961314-52ae-4ded-a522-089e3848453b.png)

Reply:
Hmm, now it works.